nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -411,9 +417,5 @@ def shutdown(self):
     def get_urls(scheme: str, resources: dict) -> (str, str):
         secure = requires_secure_connection(resources)
         if secure:
-            if use_aio_grpc():
-                scheme = "grpcs"
-            else:
-                scheme = "agrpcs"
-
+            scheme = "grpcs"
```

Comment:
Should this be agrpcs?

Reply:
No, should always be grpcs. This is the scheme to be exposed.